### PR TITLE
Fix invitation & reset password

### LINKED ISSUE #29: No release notes for 2.0.0
There are no release notes for 2.0.0.

### PR DESCRIPTION
**Pull request by djhi**
Fixes #29, #19, #16 and #11

### CODE REVIEW

**Review by slax57 — APPROVED** [review]

File: packages/ra-supabase-core/src/authProvider.ts [hunk 1]
```diff
@@ -77,6 +77,31 @@ export const supabaseAuthProvider = (
...
+                        redirectTo: `set-password?access_token=${access_token}&refresh_token=${refresh_token}&type=${type}`,
+                    };
+                }
+
+                if (process.env.NODE_ENV === 'development') {
+                    console.error(
+                        'Missing access_token or refresh_token for an invite or recovery'
+                    );
+                }
+            }
```
Comment (slax57) [inline packages/ra-supabase-core/src/authProvider.ts:103]: This whole section should be factorized between `handleCallback` and `checkAuth` IMO, to ease further maintenance of the code

**Review by slax57 — CHANGES_REQUESTED** [review]

File: packages/ra-supabase-core/src/authProvider.ts (hunk 1 above)
Comment (djhi) [reply packages/ra-supabase-core/src/authProvider.ts]: They don't do the exact same thing. One return, the other throws

Comment (djhi) [reply packages/ra-supabase-core/src/authProvider.ts]: Done

**Review by slax57 — APPROVED** [review]

**Review by slax57 — CHANGES_REQUESTED** [review]
One more thing I just thought of: since the v2.0.0 is already published, this needs to be a 2.0.1, and you need to adapt the changelog as well.

**Review by slax57 — APPROVED** [review]

**Pull request merged**